Documentation/Settings.cfg

```diff
@@ -34,3 +34,4 @@ h2document     = https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-u
 t3coreapi     = https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/
 t3tca         = https://docs.typo3.org/m/typo3/reference-tca/master/en-us/
 t3tsref       = https://docs.typo3.org/m/typo3/reference-typoscript/master/en-us/
+admpanel      = https://docs.typo3.org/c/typo3/cms-adminpanel/master/en-us/
```

Comment:
Can we make this `t3adminpanel`? This way the relation is clearer. Also one doesn't need to remember that `admin` was abbreviated as `adm`. ;-)

Reply:
Yes, I only realized that after and thought I'd see what you said. Will fix!